projects/testing-library/tests/render-template.spec.ts

```diff
@@ -44,83 +44,86 @@ class GreetingComponent {
 }
 
 test('the directive renders', async () => {
-  const component = await render('<div onOff></div>', {
+  const view = await render('<div onOff></div>', {
     declarations: [OnOffDirective],
   });
 
-  expect(component.container.querySelector('[onoff]')).toBeInTheDocument();
+  // eslint-disable-next-line testing-library/no-container
```

Comment:
Is there another/proper/better way to check if a `directive` is available in the `screen`?

Would something like: `screen.getByAttribute('[onoff]')` work?
Or maybe an Angular specific methode like `getByDirective()` ??

Reply:
Not that I know of (`getBytAttribute` doesn't exist, AFAIK), perhaps it makes sense to add a helper method that's specific to ATL.
I'm going to think about this.

Reply:
That would be really nice